doc/sphinx/source/recipes/recipe_kcs.rst

```diff
@@ -0,0 +1,141 @@
+.. _recipe_kcs:
+
+KNMI Climate Scenarios
+======================
+
+Description
+-----------
+
+This recipe implements the method described in `Lenderink et al., 2014 <https://doi.org/10.1088/1748-9326/9/11/115008>`_, to prepare the 2014 KNMI Climate Scenarios (KCS) for the Netherlands. A set of 8 global climate projections from EC-Earth were downscaled with the RACMO regional climate model. Since the EC-Earth ensemble is not readily representative for the spread in the full CMIP ensemble, this method recombines 5-year segments from the EC-Earth ensemble to obtain a large suite of "resamples". Subsequently, 8 new resamples are selected that cover the spread in CMIP much better than the original set.
+
+The original method created 16 resampled datasets:
+
+* 2 main scenarios: Moderate (M) and Warm (W) (Lenderink 2014 uses "G" instead of "M").
+* 2 'sub'scenarios: Relatively high (H) or low (L) changes in seasonal temperature and precipitation
+* 2 time horizons: Mid-century (MOC; 2050) and end-of-century (EOC; 2085)
+* 2 periods: Control (1981-2010) and future (variable)
+
+The configuration settings for these scenarios can be found in table 1 of Lenderink 2014's `supplementary data <https://iopscience.iop.org/1748-9326/9/11/115008/media/erl503687suppdata.pdf>`_.
+
+Implementation
+--------------
+
+The implementation is such that application to other datasets, regions, etc. is relatively straightforward. The description below focuses on the reference use case of Lenderink et al., 2014, where the target model was EC-Earth. An external set of EC-Earth data was used, for which 3D fields were available as well. In the recipe shipped with ESMValTool, however, the target model is CCSM4, so that it works out of the box with ESGF data only.
+
+In the first diagnostic, the spread of the full CMIP ensemble is used to obtain 4 values of a *global* :math:`{\Delta}T_{CMIP}`, corresponding to the 10th and 90th percentiles for the M and W scenarios, respectively, for both MOC and EOC. Subsequently, for each of these 4 *steering parameters*, 30-year periods are selected from the target model ensemble, where :math:`{\Delta}T_{target}{\approx}{\Delta}T_{CMIP}`.
+
+In the second diagnostic, for both the control and future periods, the 8 target model ensemble members are split into 6 segments of 5 years each. Out of all :math:`8^6` possible re-combinations of these 5-year segments, eventually 8 new 'resamples' are selected based on *local* changes in seasonal temperature and precipitation. This is done in the following steps:
```

Comment:
In the ESMValTool implementation, with CCSM4, are there also 8 target model ensemble members? If not, this might be confusing.. then you might want to add 'N target model ensemble members', which would still allow to say "N^6 possible re-combinations".